scripts/update-development-status.py

```diff
@@ -0,0 +1,112 @@
+"""
+This script updates games development status
+
+To run, install from pip:
+- pygithub
+- python-gitlab
+
+Add environment variables:
+- GH_TOKEN (see https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-token)
+- GL_TOKEN (see https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#create-a-personal-access-token)
+"""
+import os
+import re
+import yaml
+
+from pathlib import Path
+from datetime import datetime, timedelta
+from github import Github, GithubException
+from gitlab import Gitlab
+
+GH_REGEX = re.compile(r'https://github.com/([^/]+)/([^/]+)')
+GL_REGEX = re.compile(r'https://gitlab.com/([^/]+/[^/]+)')
+
+
+def main():
+    gh = Github(os.environ["GH_TOKEN"])
+    gl = Gitlab('https://gitlab.com', private_token=os.environ["GL_TOKEN"])
+
+    for filename in Path('games').iterdir():
+        if not filename.is_file() or filename.suffix != '.yaml':
+            continue
+
+        games = yaml.safe_load(open(filename, encoding='utf-8'))
+        for game in games:
+            repo_url = game.get('repo', '')
```

Comment:
should continue if there is no repo, because getting latest commit date is impossible

Reply:
Fixed